sklearn/model_selection/tests/test_search.py

```diff
@@ -730,55 +749,85 @@ def test_search_iid_param():
     # create "cv" for splits
     cv = [[mask, ~mask], [~mask, mask]]
     # once with iid=True (default)
-    grid_search = GridSearchCV(SVC(), param_grid={'C': [1, 10]}, cv=cv)
+    grid_search = GridSearchCV(SVC(), param_grid={'C': [1, 10]}, cv=cv,
+                               return_train_score=True)
     random_search = RandomizedSearchCV(SVC(), n_iter=2,
                                        param_distributions={'C': [1, 10]},
-                                       cv=cv)
+                                       cv=cv, return_train_score=True)
     for search in (grid_search, random_search):
         search.fit(X, y)
         assert_true(search.iid)
 
-        # Test the first candidate
-        cv_scores = np.array(list(search.cv_results_['split%d_test_score'
-                                                     % s][0]
-                                  for s in range(search.n_splits_)))
-        mean = search.cv_results_['mean_test_score'][0]
-        std = search.cv_results_['std_test_score'][0]
+        test_cv_scores = np.array(list(search.cv_results_['split%d_test_score'
+                                                          % s_i][0]
+                                       for s_i in range(search.n_splits_)))
+        train_cv_scores = np.array(list(search.cv_results_['split%d_train_'
+                                                           'score' % s_i][0]
+                                        for s_i in range(search.n_splits_)))
+        test_mean = search.cv_results_['mean_test_score'][0]
+        test_std = search.cv_results_['std_test_score'][0]
+
+        train_cv_scores = np.array(list(search.cv_results_['split%d_train_'
+                                                           'score' % s_i][0]
+                                        for s_i in range(search.n_splits_)))
+        train_mean = search.cv_results_['mean_train_score'][0]
+        train_std = search.cv_results_['std_train_score'][0]
 
+        # Test the first candidate
         assert_equal(search.cv_results_['param_C'][0], 1)
-        assert_array_almost_equal(cv_scores, [1, 1. / 3.])
+        assert_array_almost_equal(test_cv_scores, [1, 1. / 3.])
+        assert_array_almost_equal(train_cv_scores, [1, 1])
```

Comment:
oh, never mind, I got it. Hm alright... maybe add a comment?
Having a test for a more standard use would still be nice, but whatever.